src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMSource.java

```diff
@@ -1072,7 +1073,7 @@ public DescriptorImpl getDescriptor() {
 
     @CheckForNull
     /* package */ StandardCredentials credentials() {
-        return BitbucketCredentials.lookupCredentials(
+        return BitbucketCredentials.lookupCredentialsAndTrackUsage(
```

Comment:
it's too earlier, in this way you track same credentials multiple time on webhook event. It should be tracked only at the beginning of retrieve method